File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        for i in range(0,len(nums)+1):
            flag=0
            for j in range(0,len(nums)):
                if nums[j]==i:
                    flag=1
                    break
            if flag==0:
                return i
        

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let sum = 0
    let sumOfN=0
    let res
    for(let i=0;i<nums.length;i++){
        sum = sum + nums[i]
        sumOfN = Math.floor((nums.length*(nums.length+1)) / 2)
        res = sumOfN - sum
    }
    return res
    
};